draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -11247,15 +11266,24 @@ Content-Encoding: gzip
    URIs are intended to be shared, not secured, even when they identify secure
    resources. URIs are often shown on displays, added to templates when a page
    is printed, and stored in a variety of unprotected bookmark lists.
+   Many servers, proxies, and user agents log or display the target URI
+   in places where it might be visible to third parties.
    It is therefore unwise to include information within a URI that
    is sensitive, personally identifiable, or a risk to disclose.
 </t>
 <t>
-   Authors of services ought to avoid GET-based forms for the submission of
-   sensitive data because that data will be placed in the target URI. Many
-   existing servers, proxies, and user agents log or display the target URI
-   in places where it might be visible to third parties. Such services ought
-   to use POST-based form submission instead.
+   When an application uses client-side mechanisms to construct a target URI
+   out of user-provided information, such as the query fields of a form using
+   GET, potentially sensitive data might be provided that would not be
+   appropriate for disclosure within a URI. POST is often preferred in such
+   cases because it doesn't construct a URI; instead, POST transmits the
+   potentially sensitive data in the request body. However, this hinders
```

Comment:
```suggestion
   cases because it usually doesn't construct a URI; instead, POST of a form transmits the
   potentially sensitive data in the request body. However, this hinders
```